src/pages/home/Home.jsx

```diff
@@ -13,10 +13,9 @@ export default function Home() {
   const [crash, setCrash] = useState(false);
 
   const { data, loading, refresh } = useGetMe();
-  const unprocessedAssetGroupId = get(data, [
-    'unprocessed_asset_groups',
-    '0',
-  ]);
+  const unprocessedAssetGroups = get(data, 'unprocessed_asset_groups', [])
+  const unprocessedBulks = unprocessedAssetGroups.filter(ag => get(ag, 'uploadType') !== 'form')
```

Comment:
Same here re: future-proof. Will "not form" always mean unProccessed?

Reply:
AFAIK the answer is yes. We can always change it if necessary